Vagrantfile

```diff
@@ -10,7 +10,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "http://bit.ly/debian-wheezy-32-box"
+  config.vm.box = "https://github.com/kraksoft/vagrant-box-debian/releases/download/7.7.0/debian-7.7.0-amd64.box"
```

Comment:
This error has nothing to do with this change. You’re mixing up the Sphinx package URL with the VM URL. Apparently, Sphinx removed all their 2.2.9 from [their website](http://sphinxsearch.com/downloads/archive/) for some reason.